testsuite/features/srv_spacewalk_remove_channel_tool_delete_channel.feature

```diff
@@ -0,0 +1,46 @@
+# COPYRIGHT (c) 2019 SUSE LLC
+# Licensed under the terms of the MIT License.
+
+Feature: Test delete channels with clone using spacewalk-remove-channel tool
+  we cannot delete a channel if it  have a clone
```

Comment:
"Test delete" => "delete": the features are features of the software, not features of the testsuite.

"if it  have" => "if it has"
"  we" => "        We"

Reply:
done. thanks